pan-domain-auth-core/src/main/scala/com/gu/pandomainauth/model/AuthenticatedUser.scala

```diff
@@ -5,4 +5,5 @@ import java.util.Date
 case class AuthenticatedUser(user: User, authenticatingSystem: String, authenticatedIn: Set[String], expires: Long, multiFactor: Boolean) {
 
   def isExpired = expires < new Date().getTime
+  def hasExpiryExtension(extension: Long) = (expires + extension) > new Date().getTime
```

Comment:
I prefer grace, thanks.
The important thing here though is to have a `def` that can find out if you are in that grace period, not that you've expired it to be used in [conjunction with isExpired](https://github.com/guardian/pan-domain-authentication/pull/6/files#diff-71a4f683684a360e5cf56449b5cbe2bdR194).

I can try and work that logic to work with another `is` I just couldn't think of one.